constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/utils/TimeCycleSplineSet.java

```diff
@@ -107,7 +107,7 @@ public static class CustomSet extends TimeCycleSplineSet {
         KeyFrameArray.CustomArray mConstraintAttributeList;
         KeyFrameArray.FloatArray mWaveProperties = new KeyFrameArray.FloatArray();
         float[] mTempValues;
-        float[] mCache;
+        @SuppressWarnings("HidingField") float[] mCache;
```

Comment:
Perhaps John should double check, but I think we actually want to resolve this.
The issue is that the containing class contains a variable with the same name. I don't know how this is resolved at compile time, but it might be effectively ignored.

Seeing how it's used in `CustomVarSet` I think we should be able to rename it: `mCustomCache` (the same name used in CustomVarSet).

Reply:
Yeah, I think it makes sense! The reason I didn't rename it is that I am a bit worry about that could change some existing behaviors. I think John should have a better understanding about this.

Reply:
My CL which is merged resolved this by renaming it.
VVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVV
![image](https://user-images.githubusercontent.com/15019413/182965245-5eec5aea-5555-4f17-b785-b352b7c375b6.png)